### ISSUE TITLE
Doesn't detect first visited page

### ISSUE THREAD

**Issue by mroddev** [johannschopplich/nuxt-gtag#11]
### Environment

- Operating System: `Windows_NT`
- Node Version:     `v16.14.2`
- Nuxt Version:     `3.2.3`
- Nitro Version:    `2.2.3`
- Package Manager:  `npm@8.5.0`
- Builder:          `vite`
- User Config:      `runtimeConfig`, `modules`, `gtag`, `vite`, `css`, `build`, `nitro`, `sitemap`
- Runtime Modules:  `@pinia/nuxt@0.

### Reproduction

No reproduction.

### Describe the bug

Immediately after opening a website the first visisted page 'page_view' is not sent to Google Analytics, so it doesn't show up in Realtime or Engagement reports.
If you then start navigating to other pages it works, the problem only applies to the first page visited.

**Comment by soylomass** [comment]
Same issue

**Comment by johannschopplich** [comment]
Mmmh… That's unfortunate. Any idea why Google Analytics might not pick up the initial `page_view`?

**Comment by notflip** [comment]
Is this being looked at? I feel like this is a major bug.

**Comment by johannschopplich** [comment]
It seems that the module is correctly setting up Google Analytics, but it's not sending the initial `page_view` event. You can manually track the initial visit by sending a `page_view` event after the Google Analytics script is loaded.

```ts
// Send initial `page_view` event
useGtag('event', 'page_view', {
  page_location: window.location.href,
  page_path: useRoute().path,
  page_title: document.title,
})
```

**Comment by johannschopplich** [comment]
I have incorporated the initial `page_view` into the plugin itself. Available in [v0.5.5](https://github.com/johannschopplich/nuxt-gtag/releases/tag/v0.5.5). 
Please test it. Feedback much appreciated!

**Comment by soylomass** [comment]
Thanks for the quick fix!
In my case, sending events has never worked with this plugin (or GA4?) unless I add send_to: 'G-...' to the event parameters, so that piece of code won't work.
Has anyone experienced the same problem?

**Comment by johannschopplich** [comment]
@soylomass Are you sure the GA ID is set in your `nuxt.config.ts`? Please create a separate issue including your Nuxt configuration if that issue persists. Thanks!

**Comment by mroddev** [comment]
> I have incorporated the initial `page_view` into the plugin itself. Available in [v0.5.5](https://github.com/johannschopplich/nuxt-gtag/releases/tag/v0.5.5). Please test it. Feedback much appreciated!

Thanks for the update.
I just tested it and it still does not seem to work.. In incognito and first page load, analytics only gets the event name 'first_visit'. The 'page_view' is still not being recognised by Google  Analytics.

**Comment by johannschopplich** [comment]
@ttonyh Thanks for verifying. I believe so, too, since gtag.js will handle tracking all by itself. I will probably hide the initial `page_view` behind a module config flag. So everyone is happy. 🙂

**Comment by johannschopplich** [comment]
@ttonyh I have just released [v0.5.6](https://github.com/johannschopplich/nuxt-gtag/releases/tag/v0.5.6) with the manual tracking removed.

If another issue persists, please open a separate issue.
